src/kava.js

```diff
@@ -226,7 +227,7 @@ class Kava extends BasePlugin {
     this.eventManager.listen(this._timer, KavaTimer.Event.RESET, () => this._resetSession());
     this.eventManager.listen(this.player, this.player.Event.SOURCE_SELECTED, () => this._onSourceSelected());
     this.eventManager.listen(this.player, this.player.Event.ERROR, event => this._onError(event));
-    this.eventManager.listen(this.player, this.player.Event.FIRST_PLAY, () => this._onFirstPlay());
+    this.eventManager.listen(this.player, this.player.Event.PLAY_REQUESTED, () => this._onPlayRequested());
```

Comment:
we can use `PLAYBACK_START` event which has exactly same meaning as `FIRST_PLAY`, but just for the API request and not the video play event